addition to chapter "Commonality of Bitcoin and Lightning"

8go opened the pull request:
- as contrast to chapter of differences between Bitcoin and Lightning
- a lot is missing here
- this is just an initial draft to get the ball rolling and other contributors can add
- even if these things have been mentioned somewhere throughout the book it is useful to gather them here as a summarized list. People looking thru the TOC can find this chapter and it will give them (hopefully) a good overview of commonalities

Commit:
addition to chapter "Commonality of Bitcoin and Lightning"

- as contrast to chapter of differences between Bitcoin and Lightning
- a lot is missing here
- this is just an initial draft to get the ball rolling and other contributors can add
- even if these things have been mentioned somewhere throughout the book it is useful to gather them here as a summarized list. People looking thru the TOC can find this chapter and it will give them (hopefully) a good overview of commonalities

Commit:
added to chapter "Commonality of Bitcoin and Lightning" 

- added a few more bullet points
- more are still missing ....

Roasbeef commented on ch03.asciidoc:
```diff
@@ -556,3 +556,33 @@ When the channel is closed, the final balance of the channel is submitted to the
+
+==== Monetary Unit
+
+Both the Bitcoin network and the Lightning network use the same monetary units: bitcoins. 
```
Slight nuance here, but it's possible to pay a "milli satoshi", though this granularity of payment can't be manifested _on-chain_.

8go replied:
I added that as a distinction. I had that on the back of my mind already.

See Commit # 3.

Roasbeef commented on ch03.asciidoc:
```diff
@@ -556,3 +556,33 @@ When the channel is closed, the final balance of the channel is submitted to the
+
+=== Trustlessness
+
+Just as Bitcoin, Lightning requires the user only to trust mathematics, encryption and that the Lightning code does not have any critical bugs.
```
Lot of missing nuance here...

As an alternative, we could say something along that lines of "the security model of Lightning reduces to the security of the base blockchain.

8go replied:
Fair and correct. Your suggestion is worked in now.

See Commit # 3

Roasbeef reviewed (changes requested):
Needs a rebase as well.

Commit:
Milli-satoshis and trustlessness

- see https://github.com/lnbook/lnbook/pull/210#discussion_r420515887
- see https://github.com/lnbook/lnbook/pull/210#discussion_r420516103

8go commented:
See Commit # 3 93fea5e37f4930c2c235098666ae23992f6a1122

Commit:
Merge branch 'develop' into patch-36

8go commented:
Resolved conflicts in Commit # 4 in 93fea5e37f4930c2c235098666ae23992f6a1122

renepickhardt reviewed (approved):
I really love this section and would have written something similar at some point in time!
